src/usedrift.ts

```diff
@@ -0,0 +1,7 @@
+import { Drift } from './types.d'
+
+export const useDriftWidget = (): Drift | undefined => {
+  const drift = window.drift
+
+  return drift
```

Comment:
just for learning: doesn't this need to be `as Drift` due to the global type?

Reply:
then the return type wouldn't make sense anymore. Also it works the way it is.